watertap/data/techno_economic/component_list.yaml

```diff
@@ -118,4 +118,6 @@ viruses_enteric:
 zinc:
   constituent_longform: Zinc
 carbon_dioxide:
-  constituent_longform: Carbon Dioxide
\ No newline at end of file
+  constituent_longform: Carbon Dioxide
+foo:
```

Comment:
Why is `foo` added to this list? I don't think we should have non-real constituents in our component list.